File: mean-of-array-after-removing-some-elements/mean-of-array-after-removing-some-elements.cpp
class Solution {
public:
    double trimMean(vector<int>& arr) {
        sort(arr.begin(),arr.end());
        int n = arr.size();
        int per = (n*5)/100;
        double sum = 0;
        for(int i = per ; i < n-per ; i++){
            sum += arr[i];
        }
        int t = n - 2*per;
        return sum/t;
    }
};

File: mean-of-array-after-removing-some-elements/mean-of-array-after-removing-some-elements.py
class Solution:
    def trimMean(self, arr: List[int]) -> float:
        arr = sorted(arr)
        n = len(arr)
        sum = 0
        per =int( (n*5)/100)
        ar = arr[per:n-per]
        for i in ar:
            sum += i
        res = sum/len(ar)
        return res
